channels/us.m3u

```diff
@@ -716,7 +716,7 @@ https://playout4multirtmp.tulix.tv/live8/Stream1/playlist.m3u8
 https://granicusliveus4-a.akamaihd.net/cerritos/G0010_002/playlist.m3u8
 #EXTINF:-1 tvg-id="CGTV.us" tvg-name="CGTV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/f6dfLXA.png" group-title="Legislative",CGTV
 https://reflect-live-champaign.cablecast.tv/live/CELL-296k-234p/CELL-296k-234p.m3u8
-#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Channel 5 (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="",Channel 5 (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="Channel5.us" tvg-name="Adult Swim 24/7" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/sCRPkPk.jpg" group-title="Series",Adult Swim 24/7 [Geo-blocked]
```

Comment:
Can we copy all tags from existing Adult Swim links, please?

Reply:
Yup, I don't see why the description has been deleted.

When I added that, I made sure to precise it was an Adult Swim stream to prevent confusion. It's linked to a TV show.
All the other channels are like that.

Reply:
Sorry about that, guys.